Explorer/Assets/Scripts/Global/Dynamic/DynamicGlobalRealmNavigator.cs

```diff
@@ -32,13 +31,13 @@
 
 namespace Global.Dynamic
 {
-    public class RealmNavigator : IRealmNavigator
+    public class DynamicGlobalRealmNavigator : IRealmNavigator
```

Comment:
Why the name change? It may be a little confusing, since we don't have static neither a non global realm.